.github/workflows/build-release.yml

```diff
@@ -9,32 +9,55 @@ env:
 
 jobs:
   build_and_publish:
-    runs-on: ubuntu-latest
     environment: release
+    runs-on: ubuntu-latest
+    name: "Release: build, sign and upload the app"
+    strategy:
+      matrix:
+        php-versions: ['7.4']
+        nextcloud: ['v21.0.0beta6']
```

Comment:
Can we go for a stable version here

Reply:
switched to stable20, so we don't need to update it manually too many times